.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,10 +0,0 @@
-## What changes were proposed in this pull request?
```

Comment:
Is `PULL_REQUEST_TEMPLATE` not the right file for adding template ?

Reply:
no - see https://docs.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository. `PULL_REQUEST_TEMPLATE` should either be a directory so that there can be multiple templates, or removed entirely (and replaced with `pull_request_template.md`) as this PR does